drivers/iio/dac/ti-dac5571.c

```diff
@@ -387,15 +395,15 @@ static int dac5571_remove(struct i2c_client *i2c)
 
 #ifdef CONFIG_OF
 static const struct of_device_id dac5571_of_id[] = {
-	{.compatible = "ti,dac5571"},
-	{.compatible = "ti,dac6571"},
-	{.compatible = "ti,dac7571"},
-	{.compatible = "ti,dac5574"},
-	{.compatible = "ti,dac6574"},
-	{.compatible = "ti,dac7574"},
-	{.compatible = "ti,dac5573"},
-	{.compatible = "ti,dac6573"},
-	{.compatible = "ti,dac7573"},
+	{.compatible = "ti,dac5571", .data = (void *)single_8bit },
```

Comment:
Nitpicking, as there's no space after {, there should be none before }. I'd personally add a space in both locations, but when modifying existing drivers, it's best to stick to the existing coding style.